nemo/collections/llm/gpt/model/megatron/hyena/hyena_mixer.py

```diff
@@ -323,6 +331,9 @@ def pad_to_multiple(x, multiple=16):
             )
             z = self.mixer(x1, x2, v, _hyena_use_cp=_proj_use_cp, inference_context=inference_context)
 
-        z = rearrange(z, "b d l -> l b d").contiguous()
+        if cuhyena_rearrange is not None:
```

Comment:
@moradza are the tests passing locally? It might be a good idea to add some for `checking cuhyena_rearrange`

Reply:
@farhadrgh do you mean a single test for `cuhyena_rearrange`?

Reply:
The test was failing due to copy-paste error.

Reply:
ah ok, yeah, maybe a comparison with torch rearrange! Its a small test but it can live either here or upstream in cuhyena

Reply:
We have it in cuhyena.